mne/minimum_norm/tests/test_inverse.py

```diff
@@ -854,4 +855,24 @@ def test_make_inverse_operator_bads():
     assert (len(set(inv_['info']['bads']) - union_bads) == 0)
 
 
+@testing.requires_testing_data
+def test_inverse_ctf_comp():
+    """Test interpolation with compensated CTF data."""
+    ctf_dir = op.join(testing.data_path(download=False), 'CTF')
+    raw_fname = op.join(ctf_dir, 'somMDYO-18av.ds')
+    raw = mne.io.read_raw_ctf(raw_fname, preload=True)
+
+    events = mne.make_fixed_length_events(raw, duration=0.2)[:2]
+    epochs = mne.Epochs(raw, events, tmin=0., tmax=0.2)
+    evoked = epochs.average()
+
+    with warnings.catch_warnings(record=True):
+        noise_cov = mne.compute_covariance(epochs)
+    fwd = mne.make_forward_solution(evoked.info, None,
+                                    mne.setup_volume_source_space(pos=15.0),
+                                    mne.make_sphere_model())
+    inv = make_inverse_operator(evoked.info, fwd, noise_cov, loose=1.)
+    apply_inverse(evoked, inv, lambda2, 'dSPM')
```

Comment:
This should be a little bit faster (one source for fwd and no cov computation), it's what I was using before I had to stop:
```
    raw = read_raw_ctf(fname_ctf).crop(0, 1)
    raw.apply_gradient_compensation(1)
    sphere = make_sphere_model()
    cov = make_ad_hoc_cov(raw.info)
    src = setup_volume_source_space(
        pos=dict(rr=[[0., 0., 0.01]], nn=[[0., 1., 0.]]))
    fwd = make_forward_solution(raw.info, None, src, sphere, eeg=False)
    inv = make_inverse_operator(raw.info, fwd, cov, loose=1.)
    apply_inverse_raw(raw, inv, 1. / 9.)
```